assets/less/fauxton.less

```diff
@@ -185,15 +185,32 @@ table.databases {
   }
 }
 
-.notification-icon {
-  padding-right: 8px;
+.global-notification.alert-success {
```

Comment:
With less you could put all the alert info in the `global-notifications` and less will flatten it for you. Like this:

    .global-notification {
      alert-status {}
      alert-info
    }